### PR TITLE
preserve cookies in fetch

### PR DESCRIPTION
**Pull request by mccalluc**
Following the instructions in README does not run tests for me, right now. Looking into it...

### CODE REVIEW

**Comment by mccalluc** [comment]
(Do we know if the `text` method is ever used? Trying to track down usages...)

**Comment by mccalluc** [comment]
5 failing test, all involving "TypeError: Cannot read property 'getTrackObject' of null".

File: app/scripts/services/tile-proxy.js [hunk 1]
```diff
@@ -533,45 +528,48 @@ export const tileDataToPixData = (
...
-
-    r.send('GET', (error, data) => {
-      pubSub.publish('requestReceived', url);
-      const j = data && JSON.parse(data.response);
-      callback(error, j);
-      requestsInFlight -= 1;
-    });
+  
+  if (textOrJson === 'text') {
+    var mime = 'text/plain';
```
Comment (pkerpedjiev) [inline app/scripts/services/tile-proxy.js]: No `var` please. Use `let` or `const`.

File: app/scripts/services/tile-proxy.js [hunk 2]
```diff
@@ -533,45 +528,48 @@ export const tileDataToPixData = (
...
-      pubSub.publish('requestReceived', url);
-      requestsInFlight -= 1;
-    });
+  return fetchEither(url, callback, 'text');
 }
 
 function json(url, callback) {
   /**
-   * Send a JSON request mark it so that we can tell how many are in flight
+   * Send a JSON request and mark it so that we can tell how many are in flight
```
Comment (pkerpedjiev) [inline app/scripts/services/tile-proxy.js]: Any chance we could get some @param tags?

**Comment by mccalluc** [comment]
@pkerpedjiev : 
- I don't think the failures are related to changes here, but it'd still be nice if tests could pass.
- When this is done, we still need to pin a release, and update 2 docker images, I think. I guess let me know when there's a tag I can reference, and I'll make another PR for the docker image, unless you want to tackle that?

**Pull request merged**